app/templates/users_index.html

```diff
@@ -0,0 +1,66 @@
+{% extends 'help_desk_base.html' %}
+
+{% block body_content %}
+    <div class="custom-default row">
+        <div class="col-md-8">
+            <form id="user-info">
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <input type="text" class="form-control chosen-container chosen-format" id="user-name" placeholder="First and Last Name">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <input type="text" class="form-control chosen-container chosen-format" id="user-username" placeholder="Username">
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <input type="text" class="form-control chosen-container chosen-format" id="user-card-id" maxlength="5" placeholder="Card ID: 5 digit number on back of ID card">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <input type="button" class="btn btn-primary" id="add-row" value="Add User">
+                    </div>
+                </div>
+            </form>
+            <table id="user-table"  class="table table-striped table-bordered">
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th data-toggle="tooltip" data-placement="top" title="Student Name">Name</th>
+                        <th data-toggle="tooltip" data-placement="top" title="Student Username">Username</th>
+                        <th data-toggle="tooltip" data-placement="top" title="Student Card ID">Card ID</th>
+                    </tr>
+                </thead>
+                <tbody id="user-tbody">
+                    {% include 'users_table.html' %}
```

Comment:
Is there a reason we include the table this way? Could you just build it here and get rid of the other file?

Reply:
I guess I have it included like this so that when a student scans their ID and clocks in/out, only the table portion of the page is reloaded and not the entire page. The 'verify_scanner' method from my 'views.py' renders only 'users_table.html'. Not sure how much of a difference it makes though, or if there is a better way to be only reloading the table and not the whole page.